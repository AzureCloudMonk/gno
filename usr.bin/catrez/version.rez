/*
 * Resources for catrez version and comments
 *    Written by Dave Tribby
 */

#include ":H1:Orca:Libraries:Rinclude:Types.rez"

// Version
resource rVersion (1, $0000) {
	{1, 0, 1, release, 0},
	verUS,
	"catrez",
	"Copyright 1996 by David M. Tribby"
};


// Comment Templates

resource rComment (1, $0000) {
	"catrez v1.0.1\n"
        "Shell program to concatenate resource forks of files\n"
	"by Dave Tribby\n"
	"Copyright 1996\n\n"
        "Freeware: Distribute freely, but don't charge."
};
