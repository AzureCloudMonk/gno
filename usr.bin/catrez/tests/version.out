Usage:
	catrez [-va] -d dest_file file1 [file2 ...]
  Copyright 1996-1997 by David M. Tribby
  Version 1.0.2 (Sep 28 1997)
